src/jquery.multiselect.js

```diff
@@ -598,82 +599,78 @@
       $button
         .on({
           click: clickHandler,
-          keydown: function(e) {
-             // Change selection via up/down on a closed single select.
-				 if (!self._isOpen && !self.element[0].multiple && e.which in {38:1, 40:1} ) {  // up & down keys
-					var prev, 
-                   current = null, 
-                   next = null;
-					self.$inputs.each( function() {
-						prev = current;
-						current = next;
-						next = this;
-						if (current !== null && current.checked) {
-							return false;
-						}
-					});
-					if (e.which === 38 && prev !== null) {
-						$(prev).trigger('click');
-					}
-					else if (e.which === 40 && !next.checked) {
-						$(next).trigger('click');
-					}					 
-				 }
-				 else {
-					switch(e.which) {
-						case 27: // esc
-						case 37: // left
-						case 38: // up
-							self.close();
-							break;
-						case 40: // down
-						case 39: // right
-							self.open();
-							break;
-					}					 
-				 }
-          },
+          keydown: $.proxy(self._handleButtonKeyboardNav, self),
           mouseenter: function() {
-            if (!$button.hasClass('ui-state-disabled')) {
-              $button.addClass('ui-state-hover');
+            if (!this.classList.contains('ui-state-disabled')) {
+              this.classList.add('ui-state-hover');
             }
           },
           mouseleave: function() {
-            $button.removeClass('ui-state-hover');
+            this.classList.remove('ui-state-hover');
           },
           focus: function() {
-            if (!$button.hasClass('ui-state-disabled')) {
-              $button.addClass('ui-state-focus');
+            if (!this.classList.contains('ui-state-disabled')) {
+              this.classList.add('ui-state-focus');
```

Comment:
Cool, I didn't know this was available in IE10.

Reply:
Per https://developer.mozilla.org/en-US/docs/Web/API/Element/classList , "Basic Support" is available... Note that I am not using anything fancy w/ it.